src/_community/rules.md

```diff
@@ -46,14 +46,38 @@ Essentially, please take requests from community team members seriously and list
 something. If you have a problem with what a team member is asking you to do, we advise that you step away from the 
 situation and send a private message to another team member.
 
-#### 5) Keep all projects legal & appropriate
-
-If you need help with a project, you will not be provided with it if your project may break laws, breach a Terms of
-Service or EULA (belonging to any organisation, not just the platforms we're using), or be considered malicious or 
-inappropriate.
-
-Additionally, keep your code appropriate. Do not sexualize your code or use any slurs or other forms of derogatory
-language, either in the code itself or in the assets or filenames that make up your project.
+#### 5) Keep all projects legal, legitimate & appropriate
+
+While anyone may make use of the Quilt toolchain in accordance with the licenses applied to each project, all projects 
+developed or promoted in Quilt community spaces must meet the following criteria:
+
+* Projects must not be malicious - which includes (but is not limited to) malware, mods that include backdoors, or 
+  mods that harass specific people or groups of people (for example, by automatically kicking them from servers with 
+  the mod installed)
+* Projects must not be sexualised, hateful or otherwise inappropriate in nature, and this also applies to all code 
+  and project assets posted for any purpose
+* Projects must not break laws, including copyright laws - this includes piracy tools, as well as projects that 
+  ignore or otherwise break the licenses of other projects
+* Projects must not break the Terms of Service or EULA of any organisation, including (but not limited to) Mojang, 
+  Microsoft, Discord and any other community platforms we use
+* Projects must not hide their functionality, and should be upfront with what they do - however, projects with 
+  unfinished documentation or benign "Easter eggs" do not violate this rule
+* Projects must not give players an objective advantage over other players in a multiplayer setting, unless they
+  provide a simple, documented way for servers to disable their features - aside from the following types of projects,
+  which must have an opt-in mechanism instead:
```

Comment:
Opt-in mechanisms produce difficulty in practice where the feature in question is meant to be used on a vanilla server where it's allowed. Vanilla servers won't send custom payloads. I'm thinking specifically about the vanilla technical community, where running a vanilla server may be a hard requirement.

Reply:
This is one of the reasons I've been pushing for standardization when it comes to these mechanisms - that said, it's not like you don't have options. For example, how about datapacks, resource packs or MOTD tags?

I appreciate that this raises the skill ceiling for writing these kinds of mods, but the only way to see wide acceptance for them in the community is going to be through good-faith efforts to make sure they're not used where they shouldn't be.

Reply:
MOTD tags produce a problem with complex config systems over multiple mods, and I don't think datapacks are ideal either, but resource packs are a pretty good solution I didn't think of.

Reply:
There are definitely solutions, just... not ideal ones. Then again, I guess you could say that about many aspects of modding.

It would be interesting to see a coordinated effort to standardize this, though.

Anyway, I'm going to mark this thread as resolved for now - it might interesting to brainstorm solutions to this elsewhere though, if you have any better ideas.